### PR TITLE
feat: add governance executor

### PR DESCRIPTION
**Pull request by barrutko**

### CODE REVIEW

File: test/web3-functions/gnosis/governance-executor.test.ts [hunk 1]
```diff
@@ -0,0 +1,222 @@
...
+
+        await keeper.sendTransaction({
+            to: callData[0].to,
+            data: callData[0].data,
+        })
+
+        expect(await executor.getCurrentState(4)).to.equal(1)
+    })
+
+    it('executes multiple proposal', async () => {
```
Comment (lucas-manuel) [inline test/web3-functions/gnosis/governance-executor.test.ts]: proposals

File: test/web3-functions/mainnet/xchain-oracle-ticker.test.ts [hunk 2]
```diff
@@ -0,0 +1,199 @@
+import hre from 'hardhat'
+import { expect } from 'chai'
+import { Contract } from '@ethersproject/contracts'
+import { Web3FunctionResultCallData } from '@gelatonetwork/web3-functions-sdk/*'
+import { Web3FunctionHardhat } from '@gelatonetwork/web3-functions-sdk/hardhat-plugin'
+import { SignerWithAddress } from '@nomiclabs/hardhat-ethers/signers'
```
Comment (lucas-manuel) [inline test/web3-functions/mainnet/xchain-oracle-ticker.test.ts:6]: Can you confirm this file isn't duplicated? Not sure why its in the diff

**Review by lucas-manuel — CHANGES_REQUESTED** [review]

File: test/web3-functions/gnosis/governance-executor.test.ts (hunk 1 above)
Comment (barrutko) [reply test/web3-functions/gnosis/governance-executor.test.ts]: Fixed

File: test/web3-functions/mainnet/xchain-oracle-ticker.test.ts (hunk 2 above)
Comment (barrutko) [reply test/web3-functions/mainnet/xchain-oracle-ticker.test.ts:6]: Yes, it was. Fixed.

**Review by lucas-manuel — APPROVED** [review]

**Pull request merged**